apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -16,12 +16,13 @@ import {
 import DateCalendar from './date-calendar/date-calendar';
 import { EventPage } from './../event-page/event-page';
 import { useToast } from '../../hooks/useToast';
+import LocationInput from '../create-event/location-autocomplete/location-autocomplete-modal';
```

Comment:
You should import Components with the same name you used as the default export.
```suggestion
import LocationAutocompleteModal from '../create-event/location-autocomplete/location-autocomplete-modal';
```